### ISSUE TITLE
Availability percentage is not updating

### ISSUE THREAD

**Issue by adrianobezer** [pi-node/instructions#409]
Since i reinstall my node my availability is always the same. What could be the problem?
![WhatsApp Image 2024-12-18 at 15 51 11](https://github.com/user-attachments/assets/2033dbea-d716-42e8-9187-382e221d6ab2)
![WhatsApp Image 2024-12-11 at 11 24 04](https://github.com/user-attachments/assets/e9a27ad5-b3b3-4761-a85a-9d8769fa0fb9)

**Comment by HussinAlshaher** [comment]
also its not updated on my Node since 09/12/2024 !!!

**Comment by adrianobezer** [comment]
> also its not updated on my Node since 09/12/2024 !!!

have you had any updates yet?

**Comment by HussinAlshaher** [comment]
no, no updates till now , but i can see my node can accept 2 or 3 incoming connections currently

**Comment by Chippleadder** [comment]
This is a known issue and effecting most all nodes . The core team is working on a fix and will be available shortly. Until then any new node installation after that date will read n/a in that area. Trust there are records being maintained as to a nodes availability percentages so when fix is applied it will reflect true percentages. Best regards...

**Comment by user0482** [comment]
![Image](https://github.com/user-attachments/assets/5b3150d1-00f6-4c1b-93ec-77a75c28d550)

same isssue here, no updates for 4th day
next day it stopped giving bonuses at all
